scopes/workspace/workspace/eject-conf.cmd.ts

```diff
@@ -14,12 +14,17 @@ type EjectConfOptionsCLI = {
 
 export default class EjectConfCmd implements Command {
   name = 'eject-conf <pattern>';
-  description = 'eject components configuration (create a `component.json` file)';
+  description =
+    'eject components configuration (create a `component.json` file) - note this can be reversed at any time by snapping/tagging changes and deleting the component.json file';
```

Comment:
@itaymendel @davidfirst WDUT about this for added explanation. I feel we can't just leave as it is as it's too different to what they'll expect e.g. from comparing to ejecting a CRA application

Reply:
Technically, if you didn't change this component.json file after you ejected, you can just remove it, no need to snap/tag. 
Anyway, I'm good with this change, but put it in the `extendedDescription` prop. Otherwise, it'll be shown in `bit --help`.

Reply:
done